modules/servers/openprovider/configs.example.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+    // Custom fields name
+    'service_custom_fields_0' => 'License Number',
```

Comment:
why such keys without sense?

Reply:
This module uses for working with products and product addons in whmcs. And each of them uses email templates.
And this email templates can use custom fields that named like this:
{$service_custom_fields.0}, where first field should be license number
{$service_custom_fields.1}, where second field should be activation code

And it make sense for them